lib/auth/userloginstate/generator_test.go

```diff
@@ -554,6 +619,25 @@ func newAccessListMembers(t *testing.T, clock clockwork.Clock, accessList string
 	return alMembers
 }
 
+func newAccessListMemberWithKind(t *testing.T, clock clockwork.Clock, accessList string, kind string, member string) *accesslist.AccessListMember {
+
+	var err error
+	res, err := accesslist.NewAccessListMember(header.Metadata{
```

Comment:
the name is used to store access key member in backend

This is rare case but it can happen that a member `alice` exist where the alice is also name of access list.

That will result that the alice membership will be overwritten by the access list membership entry. 
 
What do you thing about decoupling  the name of the access list from the reference to the name of dynamic membership ?

Reply:
I think this makes sense. Am I right in saying this would only come up for manually created access list resources as I think all of them get uuids?